wp/lib/bap_wp/src/environment.ml

```diff
@@ -418,29 +419,37 @@ let get_loop_handler (env : t) :
 let get_call_preds (env : t) : ExprSet.t =
   env.call_preds
 
-let get_arch (env : t) : Arch.t =
+let get_arch (env : t) : Theory.target =
   env.arch
 
+let var_comp : (_, _) Set.comparator = (module Var.Set.Elt)
```

Comment:
What is `var_comp` supposed to do? Maybe a comment would help.

Reply:
Sure. FYI `var_comp` is the comparator for `Var` sets, which I need to cast sets of `Theory.Var.Top.t`s into `Bap.Std.Var.t`s. A bit of tedious book-keeping, really.